src/hammer-vlsi/hammer_vlsi.py

```diff
@@ -1549,14 +1574,21 @@ def append_mmmc(cmd: str) -> None:
             list=library_set_name
         ))
         # extra junk: -opcond ...
+        rc_corner_name = "rc_cond"
+        append_mmmc("create_rc_corner -name {name} -temperature {temp} -qrc_tech {qrc}".format(
+            name=rc_corner_name,
+            temp=120, # TODO: this should come from tech config
```

Comment:
What are the units of this `temp`?

Reply:
Celsius. Do you want me to note that?

Reply:
Yeah that make sense and would probably be nice to document.